ibexa/commerce/4.3/config/packages/ibexa.yaml

```diff
@@ -118,3 +118,9 @@ framework:
     esi: true
     fragments: true
     http_method_override: true
+
+    rate_limiter:
+        corporate_account_application:
+            policy: 'fixed_window'
+            limit: 1
```

Comment:
Shouldn't we increase limit per 5 minutes period? `1` seems to be a little low to me.

Reply:
This is limit per unique user (IP). I guess multiple submissions from the very same user applying for multiple different companies will be a very rare case.

Reply:
Makes sense, thanks for the clarification.